src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -44,7 +44,10 @@ function updateFunctionalComponent(current, workInProgress) {
 }
 
 function updateHostComponent(current, workInProgress) {
-  console.log('host component', workInProgress.type, typeof workInProgress.input.children === 'string' ? workInProgress.input.children : '');
```

Comment:
I’d say let’s leave this as is. This code changes super fast right now and it’s not used by real React itself anyway yet. It’ll be cleaned up before it’s actually used. For now, not worth introducing potential merge conflicts or polluting the history.